Geometry/CMSCommonData/data/dd4hep/cmsExtendedGeometry2021.xml

```diff
@@ -4,286 +4,286 @@
   <close_geometry/>
 
   <IncludeSection>
-    <Include ref="Geometry/CMSCommonData/data/materials/2021/v1/materials.xml"/>
```

Comment:
@slomeo - as we discussed it this morning, please remove this file from the PR. It's not correct as is.

Reply:
done